src/ProjectNeon/Assets/Data/Templates/CardAction.cs

```diff
@@ -14,4 +14,15 @@ public void Apply(Target target)
             effect => effect.Apply(target)
         );
     }
+
+    public Scope Scope

```

Comment:
For these properties, use lambda syntax. It makes the code more terse.

Reply:
Done